### ISSUE TITLE
NTR: Thorax flat (Chest, respiratory & breathing)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7791]
New term request Thorax flat (elthat_101205161330): Chest flattened, thoracic, decreased antero-posterior direction  UMLS: 1864447

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000327
- label: Thoracic cage flatness
- parents: [Abnormal rib cage morphology (HP:0001547)](https://hpo.jax.org/app/browse/term/HP:0001547)
- pmid: PMID:3500611
- def: The anteroposterior dimension of the chest on the lateral radiograph is often relatively narrow compared with the chest width on the frontal radiograph.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7791
- synonyms: Flat chest(NONE)
